examples/leds_shell/Makefile

```diff
@@ -0,0 +1,52 @@
+####
+#### Sample Makefile for building applications with the RIOT OS
+####
+#### The example file system layout is:
+#### ./application Makefile
+#### ../../RIOT
+####
```

Comment:
We don't have such comments in other examples. What's the reason to add them specifically here?